src/js/controller/controller.js

```diff
@@ -248,6 +248,16 @@ Object.assign(Controller.prototype, {
                         index: _model.get('item'),
                         item: playlistItem
                     });
+                    const { title, image } = playlistItem;
+                    if ('mediaSession' in navigator && title && image) {
```

Comment:
We only want to do this if we have both `title` and `image`?

Reply:
How about `(title || image)`?